pkgs/top-level/all-packages.nix

```diff
@@ -3700,6 +3700,8 @@ with pkgs;
 
   frangipanni = callPackage ../tools/text/frangipanni { };
 
+  frece = callPackage ../tools/text/frece { };
```

Comment:
text? probably something with development or sysops since it's something with databases. But it doesn't really matter

Reply:
Yeah... I was wondering where would be the ideal place for it. `development/tools/frece` maybe?